openfe/tests/protocols/test_openmm_equil_rfe_protocols.py

```diff
@@ -786,3 +788,27 @@ def test_constraints(tyk2_xml, tyk2_reference_xml):
             assert a.get('p1') == b.get('p1')
             assert a.get('p2') == b.get('p2')
             assert float(a.get('d')) == pytest.approx(float(b.get('d')))
+
+
+@pytest.mark.slow
+def test_moist_test(benzene_vacuum_system, toluene_vacuum_system,
+                    benzene_to_toluene_mapping, tmpdir):
+    # this test actually runs MD
+    # if this passes, you're 99% likely to have a good time
+    # these settings are a small sim, that has enough eq that it doesn't occaisionally crash
```

Comment:
You'll occasionally crash anyways - we see this in practice, random seeds can be problematic even when things should all be ok - I'd stick some kind of rerun fixture similar to https://stackoverflow.com/questions/52967150/can-i-retry-for-failed-tests-in-pytest